jekyll/_cci2/language-ruby.md

```diff
@@ -296,22 +302,12 @@ The `--profile` option reports the slowest examples of each run.
 
 For more on `circleci tests glob` and `circleci tests split` commands, please refer to our documentation on [Parallelism with CircleCI CLI](https://circleci.com/docs/2.0/parallelism-faster-jobs).
 
----
-
-Success! You just set up CircleCI 2.0 for a Ruby on Rails app. Check out our [Job page](https://circleci.com/gh/CircleCI-Public/circleci-demo-ruby-rails){:rel="nofollow"} to see how this looks when building on CircleCI.
```

Comment:
De-casualize the docs.

The link to the "Job page" should actually be the "Jobs" page, but I've removed it because we already link to the CircleCI project at the top of the document. Less is more!